tools/run_tests/performance/scenario_config.py

```diff
@@ -254,7 +252,7 @@ def scenarios(self):
        rpc_type='UNARY', client_type='ASYNC_CLIENT', server_type='ASYNC_SERVER',
        req_size=300, resp_size=50,
        unconstrained_client='async', outstanding=30000, channels=300,
-       offered_load=37500, num_clients=2, secure=False,
+       offered_load=37500, secure=False,
```

Comment:
Can we still keep 2 clients here to mimic the production configuration?
I tried to remove `scenario['spawn_local_worker_count'] = -1 - scenario['num_clients']` and still keep `num_clients=2` here. But I got the following failure:
```
driver.cc:251]              assertion failed: workers.size() >= num_clients + num_servers
```

Reply:
Let's submit this first if it's urgent. I will try to keep 2 clients in a follow-up PR.
But we probably should change the offered_load to 75000 if there is only one client.

Reply:
Some infrastructure to allow run_performance_tests to know to create more qps workers locally would be great here.

Reply:
(but it's an infrastructure problem - spawn_local_worker_count is the wrong tool - since this starts said processes in the qps_json_driver process, which additionally measures the wrong thing)